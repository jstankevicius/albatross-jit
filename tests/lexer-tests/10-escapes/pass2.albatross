"this \t string \t has \t tabs"
# some strings have the escape character itself
"like this \\ string"
"this is not a new line \\n right it stayed in the same line"
