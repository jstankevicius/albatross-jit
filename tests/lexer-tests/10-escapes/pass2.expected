1 1 STRING 26 this 	 string 	 has 	 tabs
1 3 STRING 18 like this \ string
1 4 STRING 58 this is not a new line \n right it stayed in the same line
